aptos-move/framework/aptos-token-objects/doc/token.md

```diff
@@ -188,34 +102,6 @@ directly understand the behavior in a writeset.
 
 
 
-<details>
-<summary>Fields</summary>
-
-
-<dl>
-<dt>
-<code>mutated_field_name: <a href="../../aptos-framework/../aptos-stdlib/../move-stdlib/doc/string.md#0x1_string_String">string::String</a></code>
```

Comment:
Do you know why these changed without changes in the corresponding Move files?

Reply:
Fixed, I accidentally uploaded the wrong ones